internal/restinterface/resthelper/client/tlshelper/tlshelper.go

```diff
@@ -35,6 +35,7 @@ var (
 	wellKnownPort map[string]string
 )
 
+//TLSHelper struct
```

Comment:
```suggestion
// TLSHelper struct
```

@nitu-s-gupta Same trivial suggestion from me. 😄

Reply:
Done